third_party/blink/renderer/brave_font_whitelist.cc

```diff
@@ -955,15 +2046,12 @@ base::span<const std::string_view> GetAdditionalFontWhitelistByLocale(
     return base::make_span(kFontWhitelistLO.begin(), kFontWhitelistLO.end());
   if (locale_language == "ml")
     return base::make_span(kFontWhitelistML.begin(), kFontWhitelistML.end());
+#endif
   return kEmptyFontSet;
 }
 
 size_t GetFontWhitelistSizeForTesting() {
-  return kFontWhitelist.size();
-}
-
-bool CanRestrictFontsForTesting() {
-  return kCanRestrictFonts;
+  return GetMainFontWhitelist().size();
```

Comment:
`!empty()`

Reply:
For the unit tests, we're actually interested in the size, not just whether the list is empty. See `third_party/blink/renderer/brave_font_whitelist_unittest.cc`